pkg/metaserver/agent/metric/provisioner/malachite/provisioner.go

```diff
@@ -130,6 +130,9 @@ func (m *MalachiteMetricsProvisioner) updateSystemStats() {
 
 func (m *MalachiteMetricsProvisioner) updateCgroupData() {
 	cgroupPaths := []string{m.baseConf.ReclaimRelativeRootCgroupPath, common.CgroupFsRootPathBurstable, common.CgroupFsRootPathBestEffort}
```

Comment:
both GeneralRelativeCgroupPaths and default cgroup path need deduplicate

Reply:
got it, I'll fix it